### PR TITLE
review01

### PR DESCRIPTION
**Pull request by netmanito**
#3

### CODE REVIEW

File: Home_new.md [hunk 1]
```diff
@@ -1,68 +1,49 @@
 # ALASTRIA #
```
Comment (marcosio) [inline Home_new.md]: This file isn't yet necessary. 

Please, delete it.

File: Links.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Alastria Links
+
+### Block Explorers
+
+* http://epe.alastria.blocksait.com/ 
+* https://alastria-explorer.councilbox.com
+
```
Comment (marcosio) [inline Links.md]: This must be a table.

File: Links.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+### NetStats
+
+* http://52.56.86.239:3000/
+* http://netstats.testnet.alastria.io.builders/
```
Comment (marcosio) [inline Links.md]: This must be a table.

File: ReinstallNode.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## Reinstall Node
+
```
Comment (marcosio) [inline ReinstallNode.md]: What is this? 

Out of scope, please delete it. 

;P

File: Home_new.md (hunk 1 above)
Comment (netmanito) [reply Home_new.md]: done

File: ReinstallNode.md (hunk 4 above)
Comment (netmanito) [reply ReinstallNode.md]: Done

File: Links.md (hunk 2 above)
Comment (netmanito) [reply Links.md]: done

File: Links.md (hunk 3 above)
Comment (netmanito) [reply Links.md]: done

**Comment by netmanito** [comment]
#4 review changes

File: Links.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+## Alastria Links
+
+### Block Explorers
+
+| URL  | Managed by|
+|:---|:---|
+| [http://epe.alastria.blocksait.com](http://epe.alastria.blocksait.com/) |  EPE |
+| [https://alastria-explorer.councilbox.com](https://alastria-explorer.councilbox.com) | CouncilBox |
+
```
Comment (netmanito) [inline Links.md]: changed to table

**Comment by netmanito** [comment]
we'll review again before pull

**Comment by netmanito** [comment]
Please review again

File: FAQ_EN.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+1. [How to pull request to alastria-node](FAQ.md#howto-pull-request-to-alastria-node)
+2. [If you have made a pull request and don't get any peers](FAQ.md#if-you-have-made-a-pull-request-and-dont-get-any-peers)
+    1. [Case 1](FAQ.md#case-1)
+    2. [Case 2](FAQ.md#case-2)
+    3. [Case 3](FAQ.md#case-3)
+    4. [Case 4](FAQ.md#case-4)
+3. [Node network_id does not match nomenclature](FAQ.md#node-network_id-does-not-match-nomenclature)
+4. [Update from branch change (network is changed)](FAQ.md#update-from-branch-change-network-is-changed)
+5. [Too many open files](FAQ.md#too-many-open-files)
+6. [Create swap file](FAQ.md#create-swap-file)
```
Comment (marcosio) [inline FAQ_EN.md]: Please, change the link if you change the title.

**Pull request merged**